tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -738,7 +738,9 @@ Then('file/folder {string} should not be marked as favorite on the webUI', async
   return client
 })
 
-Then(/there should be (\d+) files\/folders listed on the webUI/, async function(noOfItems) {
+Then(/the sum of files and folder list count should be (\d+) on the webUI/, async function(
```

Comment:
```suggestion
Then(/the count of files and folders shown on the webUI should be (\d+)/, async function(
```

Reply:
```suggestion
Then(/there should be (\d+) items listed on the webUI/, async function(
```

Can it use "items"?

Otherwise
```
Then(/the number of files and folders shown on the webUI should be (\d+)/, async function(
```

Reply:
the problem here is that the step does not count the files/folders but relies on the phoenix counter at the bottom. would be good for the step words to reflect that
![image](https://user-images.githubusercontent.com/2425577/91535018-3bd9fd80-e932-11ea-870b-4b4a700de774.png)

Reply:
OK, now I understand. @individual-it suggestion looks good.